### PR TITLE
Improve WikiToLearn section

### PR DESCRIPTION
**Pull request by ruphy**

### CODE REVIEW

File: workinggroups/_posts/2015-11-04-training.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ The HSF Training & Tutoring initiative is aimed at helping the research communit
...
 
-The basic principle of this initiative is that researchers can seldom afford to spend time writing down training material. The same applies for professors when it comes to transcribing their courses for the benefit of students in general. On the other hand, students often take notes during lessons or during their own academic research. The goal of the platform is to make easy for them to publish these notes and reports in a publicly visible site.
+The basic principle of this initiative is that researchers can seldom afford to spend time writing down training material. The same applies for professors when it comes to transcribing their courses for the benefit of students in general. On the other hand, students often take notes during lessons or during their own academic research. The goal of the platform is to make easy for them to publish these notes and reports in a publicly visible site, and to harness the power of big numbers to improve and perfect them.
```
Comment (jouvin) [inline workinggroups/_posts/2015-11-04-training.md]: I'd replace:
```
, and to harness the power of big numbers to improve and perfect them.
```

by:

```
 and to allow these materials to be improved by making each person who use it a possible contributor.
```

Comment (hegner) [inline workinggroups/_posts/2015-11-04-training.md]: "to make easy for them" -> "to make it easy for them".

**Comment by jouvin** [comment]
@ruphy please could you do the necessary updates so that we could merge this PR...

**Comment by jouvin** [comment]
@ruphy @menasce last attempt to get the problems fixed before closing the PR without merging....

**Comment by ruphy** [comment]
On mercoled� 4 maggio 2016 12:46:49 CEST Michel Jouvin wrote:
> @ruphy @menasce last attempt to get the problems fixed before closing the PR
> without merging....

Hi, I will change it in the next days. Feel free to add the changes yourself 
in the meantime.

Bye,
-Riccardo

**Comment by jouvin** [comment]
@ruphy thanks. It is much easier if you can do it. Else we have to close your PR and open a new one with your changes...

**Comment by jouvin** [comment]
@ruphy I close this PR as you don't seem to find the time to address the comments... Feel free to reopen it after fixing them...

**Pull request closed without merging**